### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by chrzrdx**
- [x] Upgrade dependencies to their latest versions

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default defineConfig({
         v3_fetcherPersist: true,
         v3_relativeSplatPath: true,
         v3_throwAbortReason: true,
+        v3_lazyRouteDiscovery: true,
```
Comment (chrzrdx) [inline vite.config.js:12]: Stabilized in Remix 2.13.0

File: biome.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.8.2/schema.json",
+  "$schema": "https://biomejs.dev/schemas/1.9.3/schema.json",
```
Comment (chrzrdx) [inline biome.json:2]: Changes based on the init file when running `biome init` on a fresh project

File: app/root.jsx [hunk 3]
```diff
@@ -53,7 +53,7 @@ export default function App() {
         <Meta />
         <Links />
       </head>
-      <body className="relative grid min-h-screen font-sans text-sand-12 antialiased before:absolute before:top-0 before:bottom-0 before:left-0 before:w-full before:bg-[url('/back.jpg')] selection:bg-amber-6 dark:before:invert-[.95] dark:before:saturate-[50%] dark:before:sepia-[.85]">
+      <body className="relative grid min-h-screen font-sans text-sand-12 antialiased selection:bg-amber-6 before:absolute before:top-0 before:bottom-0 before:left-0 before:w-full before:bg-[url(/back.jpg)] dark:before:invert-[.95] dark:before:saturate-[50%] dark:before:sepia-[.85]">
```
Comment (chrzrdx) [inline app/root.jsx:56]: The single quotes around the url broke biome's class sorting for some reason

File: app/.server/parse-markdown.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ let highlighter;
 async function parseMarkdown(filepath, { clockOffset } = { clockOffset: 0 }) {
   if (!highlighter) {
     highlighter = await createHighlighter({
-      langs: ['md', 'sh', 'bash', 'rust', 'text', 'yaml'],
+      langs: ['md', 'sh', 'rust', 'text', 'yaml'],
```
Comment (chrzrdx) [inline app/.server/parse-markdown.js:14]: We don't use bash anywhere, just sh

File: app/.server/parse-markdown.js [hunk 5]
```diff
@@ -24,9 +24,9 @@ async function parseMarkdown(filepath, { clockOffset } = { clockOffset: 0 }) {
     .use({ async: true, gfm: true })
     .use({
       renderer: {
-        code(code, lang) {
-          return highlighter.codeToHtml(code, {
-            lang: lang || 'text',
+        code(code) {
```
Comment (chrzrdx) [inline app/.server/parse-markdown.js:27]: Needed since we upgraded `shiki` to 1.22.0

**Review by chrzrdx — COMMENTED** [review]
Self review

**Review by sorokya — APPROVED** [review]
Changes lgtm. Thanks! 🙏

**Pull request merged**